pyvista/core/objects.py

```diff
@@ -507,6 +664,13 @@ def to_skybox(self):
         vtk.vtkSkybox
             Skybox if cube mapping is enabled.  Otherwise, ``None``.
```

Comment:
Are we happy with this behaviour? If I call a method called `to_skybox()` I'd expect either a skybox or an error.

Reply:
Changed behavior to raise if converting to a skybox is not possible.